src/utils/helper.js

```diff
@@ -0,0 +1,48 @@
+import jwt from 'jsonwebtoken';
```

Comment:
We have duplicate `helper.js` with the same implementation. Which is the appropriate one?

Reply:
The helper.js file has been deleted